contentcuration/contentcuration/static/js/edit_channel/utils/string_helper.js

```diff
@@ -302,10 +302,42 @@ function format_count(text, count){
   return count + " " + text + "s";
 }
 
+function get_translation(messages, message_id, data, data2, data3, data4){
```

Comment:
A utility function to translate messages dynamically. Copied over from the BaseViews method, but will clean up later